src/components/aboutItem/about-item.scss

```diff
@@ -1,67 +1,63 @@
-.about-item {
-  display: flex;
-  flex-direction: column;
-  overflow: visible;
-
-  &:not(:last-child) {
-    margin-bottom: 100px;
-  }
+.about-item{
+  /* About */
+  position: relative;
 
-  @media (min-width: $md) {
-    flex-direction: row;
-    justify-content: space-evenly;
-    align-items: center;
+  background: linear-gradient(180deg, #386A3F 3.65%, #0C4124 57.29%, #102137 100%);
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  height: fit-content ;
+}
 
-    &.right {
-      flex-direction: row-reverse;
-    }
+.about-title{
+  position: relative;
+  font-size: 56px;
+  padding-left: 7vw;
+  padding-top: 5%;
+  padding-bottom: 2vw;
+  line-height: 1.4em;
+  color: #CFE8E8;
+  mix-blend-mode: normal;
+
+  @media (max-width: $sm) {
+    font-size: 35px;
+    padding-left: 0;
+    text-align: center;
+    padding-top: 10%;
+    padding-bottom: 10%;
   }
 }
 
-.about-item__img {
-  width: 70%;
-  margin-left: auto;
-  margin-right: auto;
-  margin-bottom: 30px;
+.about-paragraph {
+  color: #CFE8E8;
+  width: 30vw;
   text-align: center;
+  font-size: 2vw;
+  @media (max-width: $sm){
+    width: revert;
+    padding-top: 5%;
+    padding-left: 13vw;
+    padding-right: 13vw;
+    font-size: 5vw;
 
-  @media (min-width: $md) {
-    width: 30%;
-    height: auto;
-    margin-bottom: 0;
-
-    &.img-0 {
-      width: 25%;
-    }
   }
 }
 
-.about-item__title {
-  @include h4-text;
-}
-
-.about-item__content {
-  width: 100%;
-
-  @media (min-width: $md) {
-    width: 60%;
+.about-footsteps{
+  width: 30vw;
```

Comment:
Would you mind changing this down to something like 17vw so that the footsteps match the size in the FAQ section below?

Reply:
I did 20vw cuse i thought 17 was too small... let me know how it looks

![image](https://user-images.githubusercontent.com/44928580/99201822-d223ff80-277a-11eb-918a-03fb9689323f.png)

Reply:
Oh yeah that looks great! Thank you!